From 3237f44ba7e4712653b64b36cc58b74d2d56d14f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ale=C5=A1=20Mat=C4=9Bj?= <amatej@redhat.com>
Date: Wed, 29 Jan 2020 09:08:34 +0100
Subject: [PATCH] Remove unused global variable UnfoObjListIt (RhBug:1793424)

This global header file variable wasn't declared using extern keyword
which was causing problems with gcc 10 (gcc 10 no longer ignores this
error).

https://bugzilla.redhat.com/show_bug.cgi?id=1793424
---
 libcomps/src/comps_objlist.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libcomps/src/comps_objlist.h b/libcomps/src/comps_objlist.h
index 0fcd48b..1d19167 100644
--- a/libcomps/src/comps_objlist.h
+++ b/libcomps/src/comps_objlist.h
@@ -34,7 +34,7 @@ typedef struct COMPS_ObjListIt COMPS_ObjListIt;
 struct COMPS_ObjListIt {
     COMPS_Object *comps_obj;
     COMPS_ObjListIt *next;
-} UnfoObjListIt;
+};
 
 
 /** COMPS_Object derivate representing category element in comps.xml structure*/
-- 
2.28.0

